pkg/bindings/mcms/timelock/types.go

```diff
@@ -419,6 +419,9 @@ const (
 	ErrorTimestamp = 2
 )
 
+// Minimum value required for the schedule call (fails with Error.InsufficientValue)
+var ScheduleCallMinValue = tlb.MustFromTON("0.012")
```

Comment:
nit: would it be better to make it a constant string, and use tlb.MustFromTON() of the constant ?

Reply:
Why would you need that const as string if this type is available. Think there's no need for another var.